.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_SMART_PROXY: '0'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5
+          bundler-cache: true
+      - name: Run rubocop
+        run: bundle exec rubocop --format github
+  test_ruby:
+    runs-on: ubuntu-latest
+    needs: rubocop
+    env:
+      SMART_PROXY_PATH: '../smart-proxy'
+    strategy:
+      fail-fast: true
+      matrix:
+        smart-proxy-branch: [develop]
+        ruby-version: [2.5, 2.7]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: theforeman/smart-proxy
+          ref: ${{ matrix.smart-proxy-branch }}
+          path: smart-proxy
+      - uses: actions/checkout@v2
+        with:
+          path: smart_proxy_ansible
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+          working-directory: smart_proxy_ansible
+      - name: Create logs dir
+        working-directory: smart_proxy_ansible
+        run: mkdir logs
+      - name: Run plugin tests
+        working-directory: smart_proxy_ansible
+        run: |
+          bundle exec rake test
```

Comment:
FYI: I already enabled testing on Jenkins so this is redundant now.

Reply:
nice work :+1: